infra/README.md

```diff
@@ -0,0 +1,20 @@
+# Corona Go Backend Infrastructure
+
+### Autoscaling Group
+
+### VPC
+
+### Redis Instance
+
+```
+cd infra/tf/redis
```

Comment:
@BrianGreenhill I don't have much knowledge about TF. But looks like now we are running TF for each component differently, is there a  way to bind all the infra together into one starting point? If yes can we do that, integrate this into existing app servers etc scripts ?

Reply:
we can orchestrate running terraform and other necessary deployment / testing scripts through a CI/CD system for example. circleci integrates well with github and has a free tier.